src/test/run-make/invalid-so/Makefile

```diff
@@ -0,0 +1,7 @@
+include ../../run-make-fulldeps/tools.mk
+
+DYLIB_NAME := $(shell rustc --crate-name foo --crate-type dylib --print file-names /dev/null)
```

Comment:
`/dev/null` doesn't exist on Windows. Would `echo | rustc ... -` work?

Reply:
Sure, and I fixed it for the test itself too.